### ISSUE TITLE
Build fails when target file makes use of the include_str macro for reading data 

### ISSUE THREAD

**Issue by johnsaigle** [sourcefrog/cargo-mutants#366]
`cargo mutants` appears to be unable to build and run a baseline when the unit test pulls in data from the filesystem via `include_str!('../../some/path/to/test/data')`.

I imagine this happens because mutants is builds itself in its own directory and that messes up path resolution.

Any tips to address this?

**Comment by sourcefrog** [comment]
Can you try building with `--in-place`?

Is the path an relative path pointing outside of the Cargo workspace tree?

**Comment by sourcefrog** [comment]
Maybe if the baseline build fails then we should point to a doc with some debugging steps, including trying `--in-place`. A start on this is in https://github.com/sourcefrog/cargo-mutants/wiki/Troubleshooting.

**Comment by johnsaigle** [comment]
> Is the path an relative path pointing outside of the Cargo workspace tree?
Yes, and yes the baseline build did fail. I'll try with --in-place, thanks!
